v2/oauth/create-an-authentication-provider.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Create an authentication provider
+hide_title: true
+---
+
```

Comment:
is this page relevant? Cause its not in the sidebar

Reply:
resolved